packages/design-system/src/components/Breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+import Breadcrumbs from './Breadcrumbs';
+
+export default {
+	component: Breadcrumbs,
+};
+
+const BreadcrumbsTemplate: ComponentStory<typeof Breadcrumbs> = args => {
+	return <Breadcrumbs {...args} />;
+};
+
+export const Basic = BreadcrumbsTemplate.bind({});
```

Comment:
I would challenge that CSF2 story here, and replace it as usual with our favorite autocomplete 😇 
```suggestion
export const Basic = <Breadcrumbs items={[ ... ]} />
```